src/services/users/index.js

```diff
@@ -1,11 +1,11 @@
 import findOne from './findOneUser';
+import hasPermission from './hasPermission';
+import createUser from './createUser';
 
 /**
  * This file handles all user services
  * To add another user service, import the service function as above,
  * Add it to the export statement
  * Remember to remove the default keyword since you are exporting two or more variables
  */
-export default {
```

Comment:
The services folder within `src/services` is for third-party/external services.
The user service should be moved to the path `src/modules/user/user.service.js`
The methods should not be contained in separate files like you have `./findOneUser.js`. Instead, let the user service module export all user service functions.